### PR TITLE
New rules pages

### PR DESCRIPTION
**Pull request by Rookeret73**
Pushing something today so we can close in a shorter cycle.
Of course there are many places where we need to improve phrasing, and structure will keep changing.

Open tasks from today's session:
1. Workspace to get its own sections. Repo handling to move to the workspace section?
2. Integrations move to their own section - "Rule Targets" / "Destinations" / "Output Targets"
3. Scripting Overview and Scripting Reference sections

### CODE REVIEW

File: docs/Rules-Intro-basics.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+id: rules-intro-basics
```
Comment (LiranLast) [inline docs/Rules-Intro-basics.md]: Keep file names lowercase- this is critical for Unix based systems :)

File: docs/rooks-info.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
+---
+id: rooks-info
```
Comment (LiranLast) [inline docs/rooks-info.md]: You might not want to change this as it's gonna break the world. Definitely not in a long PR.

File: docs/rules-intro-capabilities.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+id: rules-intro-capabilities
+title: Rule Capabilities
+sidebar_label: Rule Capabilities
+---
+
+## Repository configuration
```
Comment (LiranLast) [inline docs/rules-intro-capabilities.md]: This should be a part of the Workspaces section, no?

File: docs/rules-intro-related.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (LiranLast) [inline docs/rules-intro-related.md]: Not sure we need this page anymore

File: docs/rules-intro-status.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+id: rules-intro-status
+title: Rule Status
+sidebar_label: Rule Status
+---
+
+## Understanding Rule Status
+
+If your Rule is stuck in Pending state, it means that the Rookout SDK has not successfully applied the rule (yet), but no Error was reported.
```
Comment (LiranLast) [inline docs/rules-intro-status.md]: Let's start on a more positive tone, and explain the concepts :)

**Comment by Rookeret73** [comment]
Reverted the change to rooks-setup.
@LiranLast once you approve, i can work on part II :)

**Pull request merged**